spec.html

```diff
@@ -21851,9 +21851,9 @@ <h1>Static Semantics: Early Errors</h1>
 
     <emu-clause id="sec-static-semantics-isstrict">
       <h1>Static Semantics: IsStrict</h1>
-      <emu-grammar>ScriptBody : StatementList</emu-grammar>
+      <emu-grammar>Script : ScriptBody</emu-grammar>
```

Comment:
```suggestion
      <emu-grammar>Script : ScriptBody?</emu-grammar>
```
(I assume that's what you meant, otherwise the alg wouldn't be checking if `ScriptBody` is present.)

Reply:
Sure, though I think it'll be rendered the same.